getnyumba/getnyumba/settings.py

```diff
@@ -0,0 +1,138 @@
+"""
+Django settings for getnyumba project.
+
+Generated by 'django-admin startproject' using Django 2.1.4.
+
+For more information on this file, see
+https://docs.djangoproject.com/en/2.1/topics/settings/
+
+For the full list of settings and their values, see
+https://docs.djangoproject.com/en/2.1/ref/settings/
+"""
+
+import os
+import dj_database_url
+
+# Build paths inside the project like this: os.path.join(BASE_DIR, ...)
+BASE_DIR = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
+
+
+# Quick-start development settings - unsuitable for production
+# See https://docs.djangoproject.com/en/2.1/howto/deployment/checklist/
+
+# SECURITY WARNING: keep the secret key used in production secret!
+SECRET_KEY = '$lk5(nj@w*h$gy&=+$#r^d@-zaf5k8!k!rc@sx7dm*n5qt-hm^'
+
+# SECURITY WARNING: don't run with debug turned on in production!
+DEBUG = True
+
+ALLOWED_HOSTS = ['getnyumba.herokuapp.com']
```

Comment:
Please change this value and put it in the environment variables as well. These allowed hosts could change as the application grows.